src/System.Private.Threading/src/System/Threading/ThreadPool.cs

```diff
@@ -978,6 +974,70 @@ static private void WaitCallback_Context(Object state)
         }
     }
 
+
+    internal sealed class QueueUserWorkItemCallbackDefaultContext : IThreadPoolWorkItem
+    {
+        [System.Security.SecuritySafeCritical]
+        static QueueUserWorkItemCallbackDefaultContext() { }
```

Comment:
The empty explicit cctor is not needed for CoreRT. Could you please delete the empty explicit ctor here as well as in the regular QueueUserWorkItemCallback?